statistics/summary.csv

```diff
@@ -0,0 +1,58 @@
+9.0	0.67	140.22729088	15.0	8.0	767.9453658799999	627.718075
+10.0	0.67	67.78669122000001	15.0	5.0	562.07976922	494.293078
+11.0	1.33	107.01931317	15.0	10.0	915.5636191699999	808.5443060000001
+12.0	1.33	42.20279658	15.0	6.0	619.60268158	577.3998849999999
+13.0	2.0	35.405354	5.0	3.0	385.91833399999996	350.51297999999997
+14.0	2.0	0.0	5.0	3.0	350.51297999999997	350.51297999999997
+15.0	4.0	28.961028	5.0	4.0	406.794878	377.83385000000004
+16.0	4.0	0.0	5.0	3.0	350.51297999999997	350.51297999999997
+17.0	1.0	144.236537	10.0	5.0	681.010297	536.77376
+18.0	1.0	57.67627	10.0	4.0	504.5553	446.87903000000006
+19.0	2.0	215.228754	10.0	6.0	812.6807439999999	597.45199
+20.0	2.0	112.710494	10.0	4.0	593.289114	480.57862
+21.0	0.67	274.07969457	15.0	5.0	918.4525915700001	644.3728970000001
+22.0	0.67	152.0105246	15.0	4.0	703.8930086000001	551.8824840000001
+23.0	1.33	230.10718892000003	15.0	8.0	1099.5694389199998	869.4622499999999
+24.0	1.33	46.08986522000001	15.0	6.0	717.6477462199999	671.557881
+25.0	2.0	40.5129	5.0	4.0	292.62844	252.11554
+26.0	2.0	0.0	5.0	3.0	234.48037	234.48037
+27.0	4.0	0.0	5.0	5.0	312.23654	312.23654
+28.0	4.0	0.0	5.0	3.0	245.88268999999997	245.88268999999997
+29.0	1.0	129.89111	10.0	6.0	687.294668	557.403558
+30.0	1.0	31.49174	10.0	4.0	569.935369	538.443629
+31.0	2.0	61.05994	10.0	8.0	709.551018	648.491078
+32.0	2.0	0.0	10.0	5.0	583.119979	583.119979
+33.0	0.67	184.31509720000003	15.0	9.0	911.6760761999999	727.360979
+34.0	0.67	40.53215920000001	15.0	7.0	696.7162582000001	656.1840990000001
+35.0	1.33	124.4895162	15.0	12.0	975.1601461999998	850.67063
+36.0	1.33	0.0	15.0	8.0	679.9153190000001	679.9153190000001
+37.0	2.0	0.0	5.0	5.0	199.93463999999997	199.93463999999997
+38.0	2.0	0.0	5.0	3.0	184.70348199999998	184.70348199999998
+39.0	4.0	0.0	5.0	5.0	199.93463999999997	199.93463999999997
+40.0	4.0	0.0	5.0	3.0	184.86185	184.86185
+41.0	1.0	31.02955	10.0	9.0	425.1535	394.12395000000004
+42.0	1.0	0.0	10.0	6.0	310.283568	310.283568
+43.0	2.0	0.0	10.0	10.0	429.72496	429.72496
+44.0	2.0	0.0	10.0	5.0	293.866484	293.866484
+45.0	0.67	99.42773199999999	15.0	11.0	631.706331	532.2785990000001
+46.0	0.67	21.04860811	15.0	7.0	476.83004311	455.781435
+47.0	1.33	133.8207829	15.0	13.0	713.0641799000002	579.2433970000001
+48.0	1.33	129.79636250000001	15.0	6.0	604.1147375	474.31837499999995
+	handling	handling_cost	num_customers	num_routes	objective	routing_cost
+count	48.0	48.0	48.0	48.0	48.0	48.0
+mean	1.8333333333333333	59.54095085979167	10.0	5.666666666666667	521.8814937347917	462.34054287500004
+std	1.0955331614339288	70.51921978417995	4.125684985035174	2.636352520041483	234.91564442131374	180.92162920616232
+min	0.67	0.0	5.0	2.0	184.70348199999998	184.70348199999998
+25%	1.0	0.0	5.0	4.0	316.36063249999995	313.10455249999995
+50%	1.665	33.448547000000005	10.0	5.0	520.6361095	464.2680895
+75%	2.0	108.44210837749999	15.0	7.0	689.65006555	580.2125425
+max	4.0	274.07969457	15.0	13.0	1099.5694389199998	869.4622499999999
```

Comment:
I'm not merging this file - this is output, and shouldn't go into the repository.

Reply:
This is the way:

1. Remove the file and commit the deletion
2. Add `statistics/*.csv` to the gitignore file and commit this change.